src/engraving/dom/guitarbend.cpp

```diff
@@ -568,6 +579,24 @@ bool GuitarBendSegment::isUserModified() const
     return modified || LineSegment::isUserModified();
 }
 
+mu::draw::Color GuitarBendSegment::color() const
+{
+    if (score()->printing()) {

```

Comment:
No way to switch those off like it is possible for notes outside range, the setting that is stored in MScore::warnPitchRange?
Maybe via this:

`if (score()->printing() || !MScore::warnPitchRange) {`

Reply:
No, we're not bounding this to the same option as the pitch range. It'll probably get its own setting in staff properties instead